doc/api/crypto.md

```diff
@@ -2057,6 +2081,68 @@ Based on the recommendations of [NIST SP 800-131A][]:
 
 See the reference for other recommendations and details.
 
+### CCM mode
+
+CCM is one of the two supported [AEAD algorithms][]. Applications which use this
+mode must adhere to certain restrictions when using the cipher API:
+
+- The authentication tag length must be specified during cipher creation by
+  setting the `authTagLength` option.
+- The length of the initialization vector (nonce) must be one of 4, 6, 8, 10,
```

Comment:
This requirement is the length of auth tag not that of the iv. 
I think that we should limit the minimum size of the auth tag not to be less than 64 bytes according to NIST 800-38C.  In `B.2 Selection of the MAC Length`, it says that
```
n particular, a value of Tlen smaller than 64 should not be used unless the controlling protocol
or application environment sufficiently restricts the number of times that the decryptionverification
process can return INVALID
```

Reply:
It should add the limitation of iv and plaintext length as
```
The length of the initialization vector (N) must be from 7 to 13.
The maximum length of  plaintext is limited less than 2^(8*(15-N)) octets.
```